taskcluster/kinds/check/kind.yml

```diff
@@ -11,20 +11,22 @@ transforms:
     - taskgraph.transforms.run:transforms
     - taskgraph.transforms.task:transforms
 
+task-defaults:
+    run-on-tasks-for:
+        - "github-pull-request"
+        - "github-push"
+    attributes:
+        code-review: true
+    worker-type: b-linux
+    worker:
+        docker-image: {in-tree: 'python39'}
+        max-run-time: 1800
+    dependencies:
+        docker-image: build-docker-image-python39
+
 jobs:
     ruff:
```

Comment:
nit: it would be nice to give this a suffix (maybe `-lint`?) to distinguish this from `ruff-format`.